phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -2417,6 +2417,15 @@ public void createTable(RpcController controller, CreateTableRequest request,
                 // table/index/views.
                 tableMetadata.add(MetaDataUtil.getLastDDLTimestampUpdate(tableKey,
                     clientTimeStamp, EnvironmentEdgeManager.currentTimeMillis()));
+                if (tableType == INDEX) {
```

Comment:
@shahrs87 We also need to do this when dropping an index, right?

Reply:
Yeah, I realized that and withdrew the review ask. :P